papaparse.js

```diff
@@ -288,7 +291,10 @@ License: MIT
 			if (!_input.length || Array.isArray(_input[0]))
 				return serialize(null, _input, _skipEmptyLines);
 			else if (typeof _input[0] === 'object')
-				return serialize(objectKeys(_input[0]), _input, _skipEmptyLines);
+				return serialize(
+					_columns ? _columns : objectKeys(_input[0]),
```

Comment:
I think we can use _columns || objectKeys(input[0]) which will remove the ? and we are able to set all arguments in the same line